res/keyboard/cs_CZ.kbd.cfg

```diff
@@ -61,12 +64,18 @@ LoadSelectedTrack Shift+Left
 eject Alt+Shift+Left
 
 loop_in ě
+loop_in_goto Shift+2
 loop_out š
-reloop_exit č
+loop_out_goto Shift+3
+reloop_toggle č
+reloop_andstop Shift+4
 
-beatloop_4_toggle q
+beatloop_activate q
+beatlooproll_activate Shift+q
 loop_halve w
+beatjump_forward Shift+w
```

Comment:
Oops, ``beatjump_forward`` and ``beatjump_backward`` are assigned twice.

I`d like to  use the vacant ``Shift+q`` and ``Shift+e`` shortcuts to scale the ´´beatjump_size´´ inside the spinbox. Does this even work with the keyboard? ´´beatjump_size_up´´ and ´´beatjump_size_down´´do not work...

It works with ``loop_halve`` and ``loop_double``for the `` beatloop_size``, and it would be nice to have something similar for ´´beatjump_size´´